src/constructs/core/stack.ts

```diff
@@ -138,6 +130,24 @@ export class GuStack extends Stack implements StackStageIdentity, GuMigratingSta
 
       this.tryAddRepositoryTag();
     }
+
+    const { node } = this;
+
+    node.addValidation({
+      validate(): string[] {
+        const NO_ERRORS: string[] = [];
+
+        const isMigrating = !!node.findAll().find((construct: IConstruct) => construct instanceof CfnInclude);
+
+        if (isMigrating) {
+          Annotations.of(node.root).addWarning(
```

Comment:
How does this warning get surfaced? Is it to the console?

Reply:
> How does this warning get surfaced? Is it to the console?

Yes. Screenshot added in #1286.